terraform/testcases/hostmetrics_receiver/parameters.tfvars

```diff
@@ -0,0 +1,7 @@
+# this file is defined in validator/src/main/resources/validations
+validation_config     = "hostmetrics-metric-linux-validation.yml" #incase windows ami_family is used in ec2 then this property will be overridden in EC2
```

Comment:
Where does the responsibility of choosing the correct `validation_config` lie? In what cases would a test case require different `validation_config` files? To me it seems like if a test case requires a different `validation_config` then it should be it's own test case. 

Can you walk through the use case or constraints where it made it necessary for the terraform ec2 module to ensure the correction `validation_config` value was chosen?